### DISCUSSION TITLE
How do I use fast registration?

### DISCUSSION THREAD

**Discussion by SandraGH5** [flyteorg/flyte#1073] [Q&A] [answered]

**Comment by SandraGH5** [comment, marked as answer]
Please visit "Fast Registration" at https://docs.flyte.org/projects/cookbook/en/latest/auto/deployment/workflow/fast_registration.html.

**Reply by stephen37** [reply]
That page doesn't exist, the new one is https://docs.flyte.org/projects/cookbook/en/latest/auto/deployment/fast_registration.html.

**Reply by SandraGH5** [reply]
Thank you for updating this, Stephen!
